docs/self-managed/console-deployment/configuration.md

```diff
@@ -0,0 +1,77 @@
+| ----------------------------------- | ----------------------------------------------------------- | ---------------------------------------- |
+| `KEYCLOAK_BASE_URL`                 | Base URL for Keycloak                                       | https://example.com/auth                 |
+| `KEYCLOAK_INTERNAL_BASE_URL`        | Internal Base URL for Keycloak                              | http://camunda-platform-keycloak:80/auth |
+| `KEYCLOAK_REALM`                    | Realm for Keycloak                                          | camunda-platform                         |
+| `CAMUNDA_IDENTITY_AUDIENCE`         | Audience for Console client                                 | console                                  |
+| `CAMUNDA_IDENTITY_CLIENT_ID`        | Client Id for Console client                                | console                                  |
+| `CAMUNDA_CONSOLE_CONTEXT_PATH`      | Context path for Console                                    | console                                  |
+| `CAMUNDA_CONSOLE_CUSTOMERID`        | Unique identifier of the customer                           | `customer-id`                            |
+| `CAMUNDA_CONSOLE_INSTALLATIONID`    | Unique installation id of the current customer installation | `installation-id`                        |
+| `CAMUNDA_CONSOLE_TELEMETRY_ENABLED` | Boolean flag to enable telemetry for Console Self-Managed   | `true`                                   |
+
+Console environment variables could be set in Helm via the `console.env` key. For more details, check [Console Helm values](https://artifacthub.io/packages/helm/camunda/camunda-platform#console-parameters).
+
+## Telemetry
+
+You can enable Telemetry and usage collection to help us improve our product by sending several telemetry metrics to Camunda. The information we collect will contribute to continuous product enhancement and help us understand how Camunda is used. We do not collect sensitive information and limit data points to several metrics. For more information, you can download collected data set metrics from the Telemetry page at anytime.
+
+By enabling data collection and reporting, you can get a new page to introspect Camunda 8 components metrics. Usually accessible via Monitoring tools like Prometheus, you can now access these metrics directly in the Console.
+
+To enable Usage collection, configure the parameters described in the next section.
+
+## Configuration Parameters
+
+To enable Telemetry, the following parameters need to be configured. Camunda will provide you with the customer ID (Camunda Docker user name) needed to send telemetry data to Camunda.
+| Parameter | Description | Example value |
+| ------------------- | ----------------------------------------------------------- | ----------------- |
+| `customerId` | Unique identifier of the customer. This is also a Camunda | `customername` |
```

Comment:
I think the formatting here is a bit off for line 43 👍

Reply:
Thanks @christinaausley! @theburi I fixed the formatting of the table.